File: CPP/Sorting/Find the Sum of Encrypted Integers.cpp
class Solution {
public:
    int encrypt(int num){
        string str=to_string(num);
        sort(str.begin(),str.end());
        int n=str.size();
        string res;
        for(int i=0;i<n;i++){
            res+=str[n-1];
        }
        return stoi(res);
    }
    int sumOfEncryptedInt(vector<int>& nums) {
        int ans=0;
        for(int i=0;i<nums.size();i++){
            ans+=encrypt(nums[i]);
        }
        return ans;
    }
};

File: Java/Sorting/Find the Sum of Encrypted Integers.java
class Solution {
    private int encrypt(int num){
        String str=Integer.toString(num);
        int n=str.length();
        char[] arr=str.toCharArray();
        Arrays.sort(arr);
        String res="";
        for(int i=0;i<n;i++){
            res+=arr[arr.length-1];
        }
        return Integer.parseInt(res);
    }
    public int sumOfEncryptedInt(int[] nums) {
        int ans=0;
        for(int i=0;i<nums.length;i++){
            ans+=encrypt(nums[i]);
        }
        return ans;
    }
}
